lib/Doctrine/Persistence/Mapping/AbstractClassMetadataFactory.php

```diff
@@ -45,23 +62,54 @@ abstract class AbstractClassMetadataFactory implements ClassMetadataFactory
     /**
      * Sets the cache driver used by the factory to cache ClassMetadata instances.
      *
+     * @deprecated Deprecated in favor of setCache
```

Comment:
Do we want to use same deprecation templates as in ODM? 

> `setCacheDriver` was deprecated in doctrine/persistence 2.2 and will be removed in 3.0. Use `setCache` instead

Reply:
I'll introduce the deprecation-contracts package in a separate PR and update these messages then 👍